src/platform/packages/shared/kbn-esql-ast/src/visitor/global_visitor_context.ts

```diff
@@ -454,10 +474,23 @@ export class GlobalVisitorContext<
           }
         }
       }
+      case 'query': {
+        if (!this.methods.visitQuery || expressionNode.type !== 'query') break;
+        return this.visitQuery(parent, expressionNode, input as any);
+      }
     }
     return this.visitExpressionGeneric(parent, expressionNode, input as any);
   }
 
+  public visitQuery(
```

Comment:
What is query in this context?

Reply:
Fork can contain multiple queries. In this case, we would be visiting a subquery of `FORK`. Does that answer the question?

Reply:
Yeah I am just not sure about the `query` term. Maybe subQuery is more self explanatory? Wdyt?

Reply:
Done in https://github.com/elastic/kibana/pull/216743/commits/21a070162a5ac704bf24899e331b88280c4a5304